api/controllers/event.js

```diff
@@ -15,6 +15,8 @@ const pajda = /^[(\@all)(@here)].*[Pp][Aa][Jj][Dd][Aa]$/;
 const pobudka = /^[(\@all)(@here)].*[Pp]obudka$/;
 const maksmak = /^[(\@all)(@here)].*[Mm]aksmak$/;
 const moreish = /^[(\@all)(@here)].*[Mm]oreish$/;
+const hagiel = /^[(\@all)(@here)].*[Hh]agiel.*$/;
```

Comment:
nie dawaj .* na końcu, bo przy pytaniach o spóźnienia się będzie łapało

Reply:
ale wczesniejszy regex nakazuje aby wczesniej byl @all albo @here.
https://regex101.com/r/5qFjMR/1 tu masz przykladowy test.
Beda oczywiscie lapaly bednie takie wpisy jak `@all Hagiel nie przyjedzie`, ale takie rzeczy zdarzaja sie nadzwyczaj rzadko.

Reply:
imho powinno być tak jak do innych `/^[(\@all)(@here)].*[Hh]agiel$/;`
i ewentualnie warunkowy do Bistro. Tak Ci złapie jakiś głupie pytania czy zażalenia

Reply:
`(^[(\@all)(@here)].*[Hh]agiel$)|(^[(\@all)(@here)].*[Hh]agiel [Bb]istro$)` i to samo rajskie

Reply:
ok, done in next commit.